scheduler/core/plans/__init__.py

```diff
@@ -1,12 +1,12 @@
 # Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
 # For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
-from dataclasses import dataclass
+from dataclasses import dataclass, field
```

Comment:
It doesn't look like you're using `field` here from what I can tell? I don't see it anywhere else in the file.